WordPress/Classes/Utility/Logging/SentryStartupEvent.swift

```diff
@@ -9,7 +9,8 @@ startup time. This will block the thread. Do not use unless you're sure.
     private var errors = [String]()
 
     func add(error: NSError, file: String = #file, function: String = #function, line: UInt = #line) {
-        errors.append("\(function) (\(file):\(line)) \(error.localizedDescription) | userInfo: \(error.userInfo)")
+        let filename = (file as NSString).lastPathComponent
```

Comment:
Since we're only using this for one thing, it should be OK, but this is stripping too much information unless we ensure all our filenames are unique.

Also, the `userInfo` will contain a `NSFilePath`, with a device-specific UUID, so this won't likely help with grouping on sentry.

I think the deeper problem is that we are setting `event.message = "\(title): \(lastError)"`, and sentry isn't showing the breadcrumbs with all the error details. Otherwise, we should set a generic title and leave the error details outside of it